cmd/argo/commands/delete.go

```diff
@@ -87,5 +103,6 @@ func NewDeleteCommand() *cobra.Command {
 	command.Flags().StringVarP(&flags.labels, "selector", "l", "", "Selector (label query) to filter on, not including uninitialized ones, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 	command.Flags().StringVar(&flags.fields, "field-selector", "", "Selector (field query) to filter on, supports '=', '==', and '!='.(e.g. --field-selectorkey1=value1,key2=value2). The server only supports a limited number of field queries per type.")
 	command.Flags().BoolVar(&dryRun, "dry-run", false, "Do not delete the workflow, only print what would happen")
+	command.Flags().BoolVar(&force, "force", false, "Force delete workflows by removing finalizers")
```

Comment:
Do you happen to know why we have some flags which are in the listFlags structure and others that aren't? Maybe the listFlags are the ones that apply to all commands? Just want to make sure this shouldn't be part of that...